components/findMatch.tsx

```diff
@@ -0,0 +1,112 @@
+import React, {Component} from "react";
```

Comment:
il nome del componente mi sta dicendo più cosa deve fare in modo molto specifico. I componenti di solito sono cose visuali che fanno qualcosa di generalizzabile, ad esempio `searchBar`, `button`, che poi li puoi customizzare con le funzioni che più ti aggradano